ambry-router/src/main/java/com/github/ambry/router/OperationQuotaCharger.java

```diff
@@ -16,6 +16,7 @@
 import com.github.ambry.commons.BlobId;
 import com.github.ambry.quota.Chargeable;
 import com.github.ambry.quota.QuotaChargeCallback;
+import com.github.ambry.quota.QuotaException;
 import com.github.ambry.quota.QuotaMethod;
 import com.github.ambry.quota.QuotaResource;
 import com.github.ambry.rest.RestServiceException;
```

Comment:
nit: You might wanna remove the RestServiceException for most of the modified files since you replaced it with QuotaException.